README.rst

```diff
@@ -120,17 +122,6 @@ number, create a commit and a tag. To push this to github, use::
 
     git push --tags
 
-After that, to create the actual packages, run::
-
-    rm -rf dist
-    python setup.py sdist
-    python setup.py bdist_wheel --universal
-
-to create a source distribution and a wheel. These can then be uploaded where
-they need to be uploaded. The recommended way to do that is using `twine
-<https://pypi.python.org/pypi/twine>`_::
-
-    twine upload dist/* -r <index>
-
-Where `index` is the name of the index in your `~/.pypirc` where you want to
-upload.
+After that, Travis CI should automatically build and deploy the
+packages to PyPi. It would be helpful to monitor the Travis build so
+that errors can be fixed quickly.
```

Comment:
Where and how do you monitor this build? Do you have a direct link?

Reply:
https://travis-ci.org/sintefmath/Splipy/builds

Reply:
Added link in readme.